modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/form/web/internal/portlet/action/test/UploadFileEntryMVCActionCommandTest.java

```diff
@@ -217,6 +243,27 @@ public boolean isInMemory() {
 			null);
 	}
 
+	private MockLiferayPortletActionRequest
+			_getMockLiferayPortletActionRequest()
+		throws PortalException {
+
+		MockLiferayPortletActionRequest mockLiferayPortletActionRequest =
+			new MockLiferayPortletActionRequest();
+
+		mockLiferayPortletActionRequest.addParameter(
+			"oldFileEntryId", String.valueOf(_oldFileEntry.getFileEntryId()));
+
+		ThemeDisplay themeDisplay = new ThemeDisplay();
+
+		themeDisplay.setCompany(
+			_companyLocalService.fetchCompany(TestPropsValues.getCompanyId()));
+
+		mockLiferayPortletActionRequest.setAttribute(
+			WebKeys.THEME_DISPLAY, themeDisplay);
+
+		return mockLiferayPortletActionRequest;
+	}
```

Comment:
@icgsbr can you reuse the existing logic from `_getMockMultipartHttpServletRequest`?